slack_sdk/web/client.py

```diff
@@ -1315,6 +1315,46 @@ def chat_scheduledMessages_list(self, **kwargs) -> SlackResponse:
         """Lists all scheduled messages."""
         return self.api_call("chat.scheduledMessages.list", params=kwargs)
 
+    def conversations_acceptSharedInvite(
+        self,
+        *,
+        channel_name: str,
+        channel_id: str = None,
+        invite_id: str = None,
+        **kwargs,
+    ) -> SlackResponse:
+        """Accepts an invitation to a Slack Connect channel.
+
+        Args:
+            channel_name (str): The name of a channel, e.g. 'connectedchannel'
+            channel_id (str): Optional ID of the channel you'd like to accept
+            invite_id (str): Optional ID of the shared channel invitation
+
+            While both fields are optional, either channel_id or invite_id must be provided.
+
+        """
+        if channel_id is None and invite_id is None:
+            raise e.SlackRequestError(
+                "Either channel_id or invite_id must be provided."
+            )
+        kwargs.update({"channel_name": channel_name})
+        if channel_id:
+            kwargs.update({"channel_id": channel_id})
+        else:
+            kwargs.update({"invite_id": invite_id})
+        return self.api_call("conversations.acceptSharedInvite", json=kwargs)
```

Comment:
If there is no strong reason to use json request body here, let's use params for new ones. Although this method does not have any array arguments at this moment, we may encounter https://github.com/slackapi/python-slack-sdk/pull/1054 with future updates of the endpoint.

Reply:
Thanks for the context with #1054. I've modified `acceptSharedInvite` and `approveSharedInvite` to accept `params` instead.